ansible/roles/installation/docker/tasks/install.yml

```diff
@@ -32,10 +32,20 @@
     name: "{{ lookup('env','USER') }}"
     groups: 'docker'
     append: 'yes'
+    
+- name: Install python-pip
+  apt:
+    name: 'python-pip'
```

Comment:
Python 3 PIP is installed here https://github.com/shadow-robot/aurora/blob/master/bin/run-ansible.sh#L63 Why do you need Python 2.7 pip?

Reply:
In case this role is run remotely, so run-ansible.sh was not run locally. I had this issue with the remote-host. Ansible couldn't install docker pip module because setuptools (part of python3-pip and python-pip) were not installed. Docker pip module is required for ansible to correctly install Docker and use docker commands like docker image.